src/inmanta/protocol/methods.py

```diff
@@ -795,17 +817,22 @@ def upload_code_batched(tid: uuid.UUID, id: int, resources: dict):
 
     :param tid: The environment the code belongs to
     :param id: The id (version) of the configuration model
-    :param resource: a dict mapping resources to dicts mapping file names to file hashes
+    :param resources: a dict mapping resources to dicts mapping file names to file hashes
```

Comment:
We should be more specific about the key here. What does "mapping reosurces" mean?